Assets/GothicVR/Scripts/Player/Menu/TextureMenu.cs

```diff
@@ -0,0 +1,29 @@
+using GVR.Manager;
+using UnityEngine;
+
+namespace GVR.Player.Menu
+{
+    public class TextureMenu : MonoBehaviour
```

Comment:
TextureMenu doesn't tell me what it is for. For textures of menue? Do we need a separate class for this?

Reply:
This is just so we can properly set the textures for the main menu, but this will be removed when c_menu is implemented